### DISCUSSION TITLE
Looking for IM expert

### DISCUSSION THREAD

**Discussion by Zelfapp** [ImageMagick/ImageMagick#2694] [Consulting]
Hope I'm in the right area for posting this?

We're looking for an IM expert to generate IM scripts. For ongoing projects requiring IM.

Here's an example of a script we need developed:

- Takes a background image. In the example below, you'll see the bottle opener is the background image.
- Ability to add an image (in this example see the cat image that is less opaque) to the background image that will wrap the background image and factor in empty areas on the background image. As you can see the cat image is full bleed.
- Also, will need to add the helmet image
- And finally some text using a specific font with interline spacing.

This is just one example, but we need an IM expert on an ongoing basis. Please PM me here and we can discuss.

![image](https://user-images.githubusercontent.com/11559805/95386008-f521e080-08ab-11eb-9358-c72cdfc1996d.png)

**Comment by fmw42** [comment]
What platform/OS?  What is your version of Imagemagick?

**Reply by Zelfapp** [reply]
Scripts will be run in AWS Lambda Layer running ImageMagick 7.0.8-45 - Linux.

**Reply by fmw42** [reply]
You will need to put your email address in your profile or here so we can contact you.

**Reply by Zelfapp** [reply]
Was set to private. You should be able to contact me now.
